orocos_kdl/src/CMakeLists.txt

```diff
@@ -80,14 +80,15 @@ ENDIF()
 #####end RPATH
 
 # Needed so that the generated config.h can be used
-INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
-TARGET_LINK_LIBRARIES(orocos-kdl ${Boost_LIBRARIES})
+TARGET_INCLUDE_DIRECTORIES(orocos-kdl PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}> )
```

Comment:
This additionally needs a `$<INSTALL_INTERFACE:include>` to export an include directory in the install case.

Reply:
Isn't that what
```
install(TARGETS ...
  INCLUDES DESTINATION include
)
```
is supposed to do [here](https://github.com/orocos/orocos_kinematics_dynamics/pull/91/files#diff-671d09e2200b8363dd7d6312d86c5a3dR91)?

From the [documentation](https://cmake.org/cmake/help/latest/command/install.html):
> `INCLUDES DESTINATION`
> This option specifies a list of directories which will be added to the `INTERFACE_INCLUDE_DIRECTORIES` target property of the `<targets>` when exported by the `install(EXPORT)` command. If a relative path is specified, it is treated as relative to the `$<INSTALL_PREFIX>`.

So probably both are valid approaches to achieve the same at the end. But having both might be redundant.